conclusion.tex

```diff
@@ -11,14 +11,14 @@ \section{Conclusion}
 		\item \textbf{What algorithms and with which parameters are used to perform the actual Machine Learning?} We divided ML algorithms into categories based on the type of feedback given to the algorithm, its goal, its type and its method.
 		\begin{itemize}
 			\item \textbf{What algorithms are used to find the best parameters for the algorithms?} We listed several "workhorse" algorithms that optimize the parameters for ML algorithms, categorized in First-order techniques, second-order techniques, coordinate descent, Markov-Chain Monte Carlo and Variational inference.
-			\item \textbf{What algorithms are used to perform the Machine Learning?} For every category we listed several ML methods, with the notable mention of Artificial Neural Networks (ANNs) and Ensemble Learning.
+			\item \textbf{What algorithms are used to perform the Machine Learning?} For every category, this paper lists several ML methods, with the notable mention of Artificial Neural Networks (ANNs) and Ensemble Learning.
 		\end{itemize}
-		\item \textbf{How can we partition and distribute these algorithms across a wide variety of hardware?} To partition and distribute Machine Learning, we looked at 5 sub-research questions:
+		\item \textbf{How can these algorithms be partitioned and distributed across a wide variety of hardware?} To partition and distribute Machine Learning, there 5 sub-research questions that need to be answered:
 		\begin{itemize}
 			\item \textbf{What is the tradeoff between computation time VS communication VS accuracy?} TODO
-			\item \textbf{How to schedule and balance the workloads?} To schedule and balance the workloads, we have to take 3 important decisions: which tasks go or don't go in parallel? What is the order in which tasks will be executed? How do we ensure an even load distribution across the machines?
+			\item \textbf{How to schedule and balance the workloads?} To schedule and balance the workloads, there are 3 major considerations that need to be evaluated: which tasks go or don't go in parallel? What is the order in which tasks will be executed? How can the load be distributed evenly across the machines?
```

Comment:
Evaluated is synonymous to considered, so evaluating a consideration is a tautology. Just "there are 3 major considerations" should be fine.